x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_actions/results.js

```diff
@@ -51,7 +51,7 @@ export function ResultLinks({ jobs }) {
         <EuiToolTip position="bottom" content={openJobsInSingleMetricViewerText}>
           <EuiButtonIcon
             href={getLink('timeseriesexplorer', jobs)}
-            iconType="stats"
+            iconType="visLine"
```

Comment:
Also the icon for the Single Metric Viewer needs switching over for the results view selector:

![image](https://user-images.githubusercontent.com/7405507/90526565-74c2e700-e168-11ea-88db-81abfbdff6fd.png)

Reply:
Fixed in a54d94f.